README.md

```diff
@@ -1,17 +1,17 @@
 # Open Lyrics Database website generator
 
-This build tool compliments [lyrics.git](https://github.com/Lyrics/lyrics) to create static HTML files in `./www/`. The contents of that directory can then be uploaded to a server to become a fully functional lyrics website.
+Build static website out of [Open Lyrics Database](https://github.com/Lyrics/lyrics).
 
 
-## Requirements
+## Installing dependencies
 
     sudo apt-get install sassc
-    pip3 install pystache simple-http-server
```

Comment:
I actually liked it better the first way you had this, showing this part of the process clearly, and manually installing the packages with pip myself. imo, any developer or even end-user trying this shouldn't have any problem using pip separately from a makefile.

Reply:
Yeah, it's a tricky one. I'm thinking that `sudo apt-get install sassc` should go into `make install-dependencies`, too... just with a smart way of determining what OS and what package manager is available, checking if it's already installed, etc...

Lemme roll it back, and then make another PR with that install-dependencies Makefile rule.